src/test/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactoryTest.java

```diff
@@ -110,69 +110,69 @@ public void testCreateVariantFromVcfDeletionEmptyAlt() {
 
```

Comment:
I think we are missing a test that throws the `NonVariantException` or the `IncompleteInformationException` where trying to create a Variant

Reply:
added test for both